everest

```diff
@@ -745,6 +719,17 @@ OCAML
     magenta "Note: you *may* want to export HACL_HOME=${xpwd}/hacl-star"
 }
 
+get_vale ()
+{
+  mkdir vale
+  vale_version=$(<hacl-star/vale/.vale_version)
+  wget "https://github.com/project-everest/vale/releases/download/v${vale_version}/vale-release-${vale_version}.zip" -O vale/vale-release.zip
```

Comment:
is there any logic to skip re-downloading if the right vale version is already present?

Reply:
I think it'll be less surprising to *not* re-download in case the right version is already there; I suspect people would assume either there's been an upgrade to Vale, or that something is wrong with their setup, if Vale is re-downloaded each time.

Thanks!

Reply:
Ok, I updated the script to skip re-downloading when the expected version is already present